src/webgpu/api/operation/storage_texture/read_write.spec.ts

```diff
@@ -213,6 +213,7 @@ class F extends GPUTest {
           usage: GPUTextureUsage.RENDER_ATTACHMENT,
           format: 'rgba8unorm',
         });
+        this.trackForCleanup(dummyColorTexture);
```

Comment:
u-nit: for the future, I don't quite know why, but "dummy" is not inclusive and in general we use `placeholder` instead.